### PR TITLE
Implement alias listeners

### PR DESCRIPTION
**Pull request by katag9k**
Add aliased commands to start incidents

### What

Implements the aliases already specified in priorities.ts to allow for multiple commands to start a specified priority incident. 

### Why

This will allow alias commands to start incidents quickly. For example, ` .critical` will start a P1 incident.

### How

Current aliases specified e.g. https://github.com/Automattic/breakingbot/blob/08a4254d8424a271c72262c0baf68bcca81e7ddc/config/priorities.ts#L12 will now launch the incident

### CODE REVIEW

**Review by mhsdef — DISMISSED** [review]
I think this makes a lot of sense.

My two cent add would be, IMO, the commands of the bot are the "API for the user"; and in my experience, "small surface area" exposed in external API keeps everything maintainable and expectations consistent. So I'd strongly suggest as _few_ aliases as possible--to reinforce collective human expectations. But that's up to y'all.

While you're in here, can you nuke the explicit `.low` listener? Your generated aliases should cover that case nicely. 👍

File: config/priorities.ts [hunk 1]
```diff
@@ -1,15 +1,14 @@
 import type { PriorityConfig } from "./types.js";
 
 export const priorityConfig: PriorityConfig = {
-	default: 2,
-	defaultLow: 3,
+	default: 3,
```
Comment (mhsdef) [inline config/priorities.ts:4]: Could we change this (`default`) in the VIP config specifically rather that Core? Or is there a complication that I'm overlooking in that?

Comment (katag9k) [reply config/priorities.ts:4]: You mean leaving default at 2?

Comment (katag9k) [reply config/priorities.ts]: Done, I forgot that this is a core setting.

File: config/priorities.ts [hunk 2]
```diff
@@ -20,13 +19,14 @@ export const priorityConfig: PriorityConfig = {
 			},
 			reportRequired: true,
 			reviewRequired: true,
+			isHighPriority: true,
```
Comment (mhsdef) [inline config/priorities.ts:22]: This is a much better, flexible, devex/ux. Thanks for solving this!

File: config/priorities.ts [hunk 3]
```diff
@@ -37,13 +37,14 @@ export const priorityConfig: PriorityConfig = {
...
 		},
 		3: {
 			name: "P3",
 			emoji: "dash",
 			description:
 				"Something is broken, or not fully working as intended, but it's not resulting in customer-facing errors or is impacting a singular or very small segment of customers. It's below “urgent”, but it needs to be fixed with high priority. Security incidents involving a _single_ customer fall here. Most plugin vulnerability incidents also belong here.",
-			aliases: ["lo", "low", "lite", "light"],
+			aliases: ["medium"],
```
[the remark this answers is not in the document]
Comment (katag9k) [reply config/priorities.ts]: done

File: config/priorities.ts (hunk 1 above)
Comment (katag9k) [reply config/priorities.ts]: done

**Review by mhsdef — APPROVED** [review]
Great work! 🚢

**Pull request merged**